lighthouse-core/audits/byte-efficiency/offscreen-images.js

```diff
@@ -134,8 +134,8 @@ class OffscreenImages extends ByteEfficiencyAudit {
       });
 
       const headings = [
-        {key: 'preview', itemType: 'thumbnail', text: ''},
-        {key: 'url', itemType: 'url', text: 'URL'},
+        {key: 'preview', itemType: 'thumbnail', text: '', styleClass: 'preview'},
+        {key: 'url', itemType: 'url', text: 'URL', styleClass: 'url'},
```

Comment:
@brendankenny convinced me that this could feel repetitive and unclear for contributors.

@patrickhulce wdyt, about `renderTable` having a switch statement in it that decides which column class to apply based on the `details.type`. Right now we don't have complicated styles with hybrid outputs. I think that would be ok for now.